backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/UserService.cs

```diff
@@ -48,6 +48,21 @@ private async Task<User> GetUserById(long id)
             return await _context.Users.FirstOrDefaultAsync(u => u.Id == id) ?? throw new KeyNotFoundException("User doesn't exist");
         }
 
+        public async Task<UserDto> CreateUserPreferences(NewUserDto userDto)
+        {
+            var newUser = _mapper.Map<NewUserDto, User>(userDto);
+
+            var existingUser = await _context.Users.FirstOrDefaultAsync(u => u.Email == userDto.Email);
+            if (existingUser is not null)
+            {
+                return _mapper.Map<User, UserDto>(existingUser);
+            }
```

Comment:
I believe if we create a user we can omit this check
And just throw an error that user with such email already exists

Reply:
If user registers via google again, he will see this error. This is the reason why I have added this check. I just remember that on different websites if user try to register again with google, he will see no error and just logins

Reply:
So, should I remove it or leave?

Reply:
for now, we don't need this, and for now we will think that this is two different users

Reply:
Okay, start fixing)

Reply:
Fixed